src/components/UserBackgroundImage.js

```diff
@@ -9,7 +9,12 @@ import SetBackgroundDailyImageMutation from 'src/utils/mutations/SetBackgroundDa
 
 dayjs.extend(isToday)
 const useStyles = makeStyles(() => ({
+  '@keyframes fadeIn': {
+    from: { opacity: 0 },
+    to: { opacity: 1 },
+  },
   background: {
+    animation: '$fadeIn 0.5s ease',
```

Comment:
Let's move the animation to the root element so that it fades in the "tint" component, too.